spec/models/registration_system_spec.rb

```diff
@@ -105,4 +107,134 @@
       expect(RegistrationSystem.verify_credentials).to be_truthy
     end
   end
+
+  describe ".update_rhsm_conf_queue" do
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+      @proxy_args = {:registration_http_proxy_server   => "192.0.2.0:myport",
+                     :registration_http_proxy_username => "my_dummy_username",
+                     :registration_http_proxy_password => "my_dummy_password"}
+
+      @miq_region = FactoryGirl.create(:miq_region, :region => 1)
+      allow(MiqRegion).to receive(:my_region).and_return(@miq_region)
+    end
+
+    it "does not modify original arguments" do
+      cloned_creds = @proxy_args.clone
+      RegistrationSystem.update_rhsm_conf_queue(@proxy_args)
+      expect(@proxy_args).to eq(cloned_creds)
+    end
+
+    it "validate that a message was queued for each server in the region" do
+      RegistrationSystem.update_rhsm_conf_queue(@proxy_args).each do |server|
+        expect(MiqQueue.exists?(:server_guid => server.guid,
+                                :class_name  => "RegistrationSystem",
+                                :method_name => "update_rhsm_conf")).to be_truthy
+      end
+    end
+
+    it "validate that one queue item was created for each server in the region" do
+      RegistrationSystem.update_rhsm_conf_queue(@proxy_args)
+      expect(MiqQueue.count).to eq(2)
```

Comment:
Keep if you want. But this seems like a duplicate of the previous spec.
I'd just move the expect clause up to the previous spec and remove this one.